mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -3265,6 +3288,20 @@ LogicalResult ExtractStridedSliceOp::verify() {
   if (getResult().getType() != resultType)
     return emitOpError("expected result type to be ") << resultType;
 
+  unsigned idx = 0;
+  for (unsigned ub = sizes.size(); idx < ub; ++idx) {
```

Comment:
nit: the ind/ub var initialization is a bit unusual, why not?
```suggestion
  for (unsigned idx = 0; idx < sizes.size(); ++idx) {
```

(same applies to insert equivalent)

Reply:
Yeah, this is not needed here.